### ISSUE TITLE
Motion Turn Off feature

### ISSUE THREAD

**Issue by neshala** [YinHangCode/homebridge-mi-aqara#66]
With iOS 11 there is option to turn off light for example if there is no motion for example in five minutes. It is disabled for mi-aqara plugin with xiaomi motion sensor. What is needed to make this available, homebridge update or this plugin update?

![img_0554](https://user-images.githubusercontent.com/1878392/30665616-d98f7c6e-9e51-11e7-86bf-bc10b7766134.jpg)

**Comment by theo-69** [comment]
I am using domoticz-homebridge-mi-gateway and its working...

**Comment by neshala** [comment]
I have exactly [this one](https://www.aliexpress.com/item/2015-Newest-Original-Xiaomi-Smart-Home-Kit-Switch-Human-Body-Security-Sensor-Door-Window-Sensor/32512721981.html?spm=a2g0s.9042311.0.0.cZWZ8l)

**Comment by theo-69** [comment]
@neshala I do not use this plugin, i use domoticz for some Xiaomi Devices

**Comment by iCarl** [comment]
This works for me

**Comment by youri50** [comment]
I have the same problem, it dont work for me.

**Comment by youri50** [comment]
It still don’t work, is there a solution?

**Comment by neshala** [comment]
I have workaround which allows me to control lights via Siri, but for motion detection + lights i am using MiHome app.

Here is how i do it step by step:

1. Add YeeLight bulb to official Yeelight app, and there i enable "Lan access" in settings for each bulb.
2. I remove bulb from Yeelight app and add it to MiHome app.
3. I add motion sensors to MiHome app as well.
4. In MiHome app i setup automation with no motion for X minutes to turn off light but at the same time i can control lights from Apple HomeKit via HomeBridge so i can say  "Lights off" and have auto off in no motion for 2 minutes.

**Comment by youri50** [comment]
Yes that can work with yeelight, bud when you want to control philips hue light with the motion sensor of Xiaomi you can't make it work :(

**Comment by viDU85** [comment]
To set up automations you need a HomePod, an Apple TV 4K or Apple TV (4th generation), or an iPad as a home hub.
